README.md

```diff
@@ -1,2 +1,6 @@
 # hello-world2
-Project to practice git skills
+Project to practice git skills 
+
+However, it is becoming clear that the workflows which can 
```

Comment:
@rjwil35  Please specify which workflow is essential to fast microservices development and I will include your updates.